modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/constants/CommerceProductFDSNames.java

```diff
@@ -27,6 +27,9 @@ public class CommerceProductFDSNames {
 	public static final String PRODUCT_CONFIGURATION_LISTS =
 		CPPortletKeys.CP_CONFIGURATION_LISTS + "-productConfigurationLists";
 
+	public static final String PRODUCT_CONFIGURATIONS =
```

Comment:
why is not PRODUCT_CONFIGURATION_ENTRIES? I think we are using the name productConfiguration (for the entries) only in the API